client/src/pages/HouseName/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+import {
+  HouseNameText,
+  Housemate,
+  HouseMates,
+  HouseNameContainer,
+  HouseNameData,
+} from './styled'
+
+import { useHouses } from '../../providers/Houses'
+import Button from './Button'
+
+const HouseName = () => {
+  const { allUsersHouses } = useHouses()
+  const location = useLocation()
+  const [viewingHouse, setViewingHouse] = useState(location.state)
+  const [userHouseNames, setUserHouseNames] = useState([])
+
+  const totalHouseMates = viewingHouse['users'].length
+
+  function getHouseName(allUsersHouses) {
```

Comment:
If you're trying to pluck properties off of an array of objects, the functional progrmaming way of doing this would be to use map, becuase map returns an array by default.

```
  function getHouseNames(allUsersHouses) {
    return allUsersHouses.map((house) => house.name)
  }
```